includes/api/class-edd-api.php

```diff
@@ -1604,6 +1605,7 @@ public function get_discounts( $discount = null ) {
 				$discount_list['discounts'][0]['product_requirements']       = edd_get_discount_product_reqs( $discount );
 				$discount_list['discounts'][0]['requirement_condition']      = edd_get_discount_product_condition( $discount );
 				$discount_list['discounts'][0]['global_discount']            = edd_is_discount_not_global( $discount );
+				$discount_list['discounts'][0]['excluded_products'] 	     = edd_get_discount_excluded_products( $discount->ID );
```

Comment:
```suggestion
				$discount_list['discounts'][0]['excluded_products']          = edd_get_discount_excluded_products( $discount->ID );
```
(Just adjusting mid-line alignment to use spaces.)